pycytominer/__about__.py

```diff
@@ -1,3 +1,4 @@
 __project__ = "pycytominer"
 __version__ = "0.2.0"
 __license__ = "BSD 3-Clause License"
+__author__ = "Broad Institute of MIT, Harvard, and Pycytominer Contributors"
```

Comment:
```suggestion
__author__ = "Pycytominer Contributors"
```

Good to know that removing this breaks sphinx. Thanks for adding it back! I originally had it just as myself, but as the project grew obviously this was no longer true.

I like simply stating pycytominer contributors here. There are many outside-the-Broad contributors.

Reply:
Thank you for the clarification here - understood! I'll add this to a batch of changes.